### PR TITLE
Feature/account lockout policy

### PR DESCRIPTION
**Pull request by rochamarcelo**
Ref: https://github.com/CakeDC/users/issues/1081

### CODE REVIEW

**Comment by rochamarcelo** [comment]
In progress

File: src/Identifier/PasswordLockout/LockoutHandler.php [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+    {
+        $lockoutField = $this->getConfig('userLockoutField');
+        $userLockoutTime = $identity[$lockoutField] ?? null;
+        if ($userLockoutTime) {
+            if (!$this->checkLockoutTime($userLockoutTime)) {//Still locked?
+                return false;
+            }
+        }
+        $timeWindow = $this->getTimeWindow();
+        $attemptsCount = $this->getAttemptsCount($identity['id'], $timeWindow);
```
Comment (steinkel) [inline src/Identifier/PasswordLockout/LockoutHandler.php:68]: array key exists

Comment (steinkel) [reply src/Identifier/PasswordLockout/LockoutHandler.php:68]: please check similar items in this file too

**Review by steinkel — CHANGES_REQUESTED** [review]
minor comment, OK to merge when fixed, thanks!

**Pull request merged**